public/scripts/world-info.js

```diff
@@ -3858,6 +3864,12 @@ export async function checkWorldInfo(chat, maxContext, isDryRun) {
                 continue;
             }
 
+            if (buffer.getExternallyActivated(entry)) {
```

Comment:
I don’t mind this being moved, personally. But it breaks the character v3 specs. Decorators need to be parsed and used before any states, as far as I know.

Reply:
I don't think there's a sane behavior in this case other than rewriting everything so that decorators are checked for after any potential entry mods, so I might as well move it down for spec adherence.